lib/matplotlib/scale.py

```diff
@@ -1,10 +1,15 @@
 """
 Scales define the distribution of data values on an axis, e.g. a log scaling.
-
-They are attached to an `~.axis.Axis` and hold a `.Transform`, which is
-responsible for the actual data transformation.
+They are defined as subclasses of `ScaleBase`.
 
 See also `.axes.Axes.set_xscale` and the scales examples in the documentation.
+
+See :doc:`/gallery/scales/custom_scale` for a full example of defining a custom
+scale.
+
+Matplotlib also supports non-separable transformations that operate on both
+axis at the same time.  They are known as projections, and defined in
```

Comment:
Grammatically, this should be axes, but that's confusing, so `Axis` directions, or something similar?

Reply:
yup